src/js/helpers/searchIndexBuilder.js

```diff
@@ -0,0 +1,73 @@
+const searchIndexBuilder = function(pages) {
+
+  const searchIndex = []
+
+  pages.forEach( page =>{
+
+    if (page.node.janisUrls.length > 0) {
+
+      const janisPages = [
+        "locationpage",
+        "eventpage",
+        "departmentpage",
+        "topiccollectionpage"
+      ]
+
+      const janisBasePagesWithTopics = [
+        "servicepage",
+        "guidepage",
+        "informationpage",
+        "officialdocumentpage",
+        "formcontainer"
+      ]
```

Comment:
I'd thought about combining these. But, that would make the conditional logic below really messy...